### PR TITLE
Init following provider template

### PR DESCRIPTION
**Pull request by rionastokes**
List and load functions for sites, collections, items OK

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -24,22 +23,22 @@ jobs:
         os: [ubuntu-latest, windows-latest, macos-latest]
         node-version: [lts/*, 17.x, 16.x]
```
Comment (rjrodger) [inline .github/workflows/build.yml:24]: check latest node versions
remove old odd versions

File: README.md [hunk 2]
```diff
@@ -1,35 +1,28 @@
-![Seneca Tangocard-Provider](http://senecajs.org/files/assets/seneca-logo.png)
+![Seneca Webflow-Provider](http://senecajs.org/files/assets/seneca-logo.png)
```
Comment (rjrodger) [inline README.md:1]: check with seneca-maintain

File: package.json [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
   ],
   "author": "Richard Rodger - richardrodger.com",
+  "contributors": [
+    "Ríona Stokes https://github.com/rionastokes",
+    "Richard Rodger - richardrodger.com"
+  ],
   "repository": {
     "type": "git",
-    "url": "git://tangocard.com/senecajs/seneca-tangocard-provider.git"
+    "url": "git://webflow.com/senecajs/seneca-webflow-provider.git"
```
Comment (rjrodger) [inline package.json:22]: fix to working github url

File: src/tsconfig.app.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (rjrodger) [inline src/tsconfig.app.json:1]: remove

File: src/tsconfig.spec.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (rjrodger) [inline src/tsconfig.spec.json:1]: remove

File: test/live.js [hunk 6]
```diff
@@ -1,12 +1,10 @@
-
 // IMPORTANT: assumes node-fetch@2
```
Comment (rjrodger) [inline test/live.js:1]: update and get working as simple smoke test

**Review by rjrodger — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (rionastokes) [reply .github/workflows/build.yml]: 5349c3f

Should remove 16.x or no?

File: README.md (hunk 2 above)
Comment (rionastokes) [reply README.md:1]: 91ff016

Comment (rionastokes) [reply README.md:1]: exist_dist failing when it shouldn't - issue logged on @seneca/maintain

File: package.json (hunk 3 above)
Comment (rionastokes) [reply package.json]: 6dc0557

File: src/tsconfig.app.json (hunk 4 above)
Comment (rionastokes) [reply src/tsconfig.app.json]: e4e19d2

File: src/tsconfig.spec.json (hunk 5 above)
Comment (rionastokes) [reply src/tsconfig.spec.json]: e4e19d2

File: test/live.js (hunk 6 above)
Comment (rionastokes) [reply test/live.js]: 19011ca

Comment (rionastokes) [reply test/live.js]: Need clarification on how/when to run file

File: package.json [hunk 7]
```diff
@@ -40,7 +45,8 @@
     "node": ">=14"
   },
   "dependencies": {
```
Comment (rjrodger) [inline package.json:47]: move to devDep

File: test/quick.js [hunk 8]
```diff
@@ -1,35 +1,15 @@
-
-
-
-const Webflow = require("webflow-api");
+const Webflow = require('webflow-api')
+const token = require('./local-env').WEBFLOW_ACCESSTOKEN
```
Comment (rjrodger) [inline test/quick.js:2]: capitalize

**Review by rjrodger — CHANGES_REQUESTED** [review]

**Pull request merged**